omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -91,7 +107,15 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 		
 		T delegate = handler.newDelegate();
 		setConvertedProperties(delegate, propertiesToCreate, handler.getCreatableProperties(), true);
-		ValidateUtil.validate(delegate);
+		
+		try {

```

Comment:
You actually just answered yourself and confirmed what i was trying say, the webservices module's framework already catches validation exceptions, wraps and sends back as part of the response with the appropriate error code as seen [here](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/BaseRestController.java#L80),  in the adminUI module all you did was to detect the same error code and the messages send by this logic in BaseRestController.validationExceptionHandler, does this make sense now?